scheduler/pkg/agent/rproxy_grpc_test.go

```diff
@@ -123,6 +126,44 @@ func TestReverseGRPCServiceSmoke(t *testing.T) {
 		return client.ModelInfer(ctx, &v2.ModelInferRequest{ModelName: dummyModelNamePrefix}) // note without suffix
 	}
 
+	numStreamMessages := 10
+	doStreamInfer := func(modelSuffixInternal, modelSuffix string) ([]*v2.ModelInferResponse, error) {
+		logger := log.New()
+		logger.SetLevel(log.DebugLevel)
+		logger.Debug("Starting stream infer")
+
+		client := v2.NewGRPCInferenceServiceClient(conn)
+		ctx := context.Background()
+		ctx = metadata.AppendToOutgoingContext(ctx, util.SeldonInternalModelHeader, dummyModelNamePrefix+modelSuffixInternal, util.SeldonModelHeader, dummyModelNamePrefix+modelSuffix)
+
+		stream, err := client.ModelStreamInfer(ctx)
+		if err != nil {
+			logger.Errorf("Error creating stream: %v", err)
+			return nil, err
+		}
+
+		responses := []*v2.ModelInferResponse{}
+		for i := 0; i < numStreamMessages; i++ {
```

Comment:
I think this is only testing the happy path ie. when there are not errors. perhaps in grpc stream there are more subtle issues when errors occur perhaps worth adding to testing.